test/src/main/java/org/apache/accumulo/test/fate/meta/ZooMutatorIT.java

```diff
@@ -79,9 +80,11 @@ public class ZooMutatorIT extends WithTestNames {
    */
   @Test
   public void concurrentMutatorTest() throws Exception {
-    File newFolder = tempDir.toPath().resolve(testName() + "/").toFile();
-    assertTrue(newFolder.isDirectory() || newFolder.mkdir(), "failed to create dir: " + newFolder);
-    try (var testZk = new ZooKeeperTestingServer(newFolder); var zk = testZk.newClient()) {
+    Path newFolder = tempDir.resolve(testName() + "/");
+    if (!Files.isDirectory(newFolder)) {
+      Files.createDirectories(newFolder);
+    }
```

Comment:
could add similar assertion: `assertTrue(Files.isDirectory(dir))` or move to util method I mentioned earlier